src/Repositories/RedisKeys.php

```diff
@@ -25,6 +25,19 @@ private function queueKey($queue, $prefix = '')
         );
     }
 
+    private function fairSignalKey($queue)
+    {
+        $signal_key_prefix_for_horizon = config('fair-queue.signal_key_prefix_for_horizon');
+        $horizon_prefix = config('horizon.prefix');
+
+        return sprintf(
```

Comment:
Can you leave an example output as a comment?

Reply:
Assume:
```
$queue = 'default'; 
$signal_key_prefix_for_horizon = config('fair-queue.signal_key_prefix_for_horizon'); // "fq:"
$horizon_prefix = config('horizon.prefix'); // "horizon:"
```

Output:
```
horizon:fq:default:[0-9]*
```

It's a key name pattern of fair signals for a specific queue